### ISSUE TITLE
Travis stage `deploy snapcraft` is failing

### ISSUE THREAD

**Issue by kishansagathiya** [ipfs/ipfs-cluster#554]
#### Pre-check

* [x] This is not a IPFS Cluster website content issue ([file those here](https://github.com/ipfs/ipfs-cluster-website/issues))
* [x] I read the [troubleshooting section](https://cluster.ipfs.io/documentation/troubleshooting/) of the website and it did not help
* [x] I searched for similar issues in the repo without luck
* [x] All my peers are running the same cluster version
* [x] All my peers are configured using the same cluster `secret`

#### Basic information

* [x] Version information (mark as appropiate):
  * [ ] Master
  * [ ] Release candidate for next version
  * [ ] Latest stable version
  * [ ] An older version I should not be using
* [x] Type (mark as appropiate):
  * [ ] Bug
  * [ ] Feature request
  * [ ] Enhancement
* [x] Operating system (mark as appropiate):
  * [ ] Linux
  * [ ] macOS
  * [ ] Windows
  * [ ] Other: `which?`
* [x] Installation method (mark as appropiate):
  * [ ] Binaries from dist.ipfs.io
  * [ ] Built from sources
  * [ ] Docker
  * [ ] Snap
  * [ ] Other: `which?`

#### Description
Travis stage `deploy snapcraft` is failing with following error
```
No valid credentials found. Have you run "snapcraft login"?
Invalid credentials: Root macaroon not in the config file. Have you run "snapcraft login"?
The command "docker run -v $(pwd):$(pwd) -t snapcore/snapcraft sh -c "apt update -qq && cd $(pwd) && snapcraft && snapcraft push *.snap --release edge"" exited with 2.
```

**Comment by hsanjuan** [comment]
It works well: https://travis-ci.org/ipfs/ipfs-cluster/jobs/434140729

It may not work in a different fork other than the original repo though (as intended).

**Comment by lanzafame** [comment]
@hsanjuan I haven't looked at travis docs yet but is there a way to run the `deploy snapcraft` stage only on merge or new tag?

**Comment by kishansagathiya** [comment]
Since my PR doesn't deal with anything snapcraft related and the error message is about not having credentials, I thought this would probably not be something that could because of my PR. And plus credentials are something that we would not keep in our source code, I thought everyone would be facing it.
I have re-ran that stage multiple times, but let me try once more anyways.

**Comment by hsanjuan** [comment]
Hmm ok I see it tries to deploy also in every PR to master. Need to update the travis file.

**Comment by hsanjuan** [comment]
Fixed in #565 etc
